### ISSUE TITLE
Navigation Bar isn't responsive for mobile screens

### ISSUE THREAD

**Issue by Git-Harshit** [many-passwords/many-passwords.github.io#5]
The current version of the Navigation Bar used in homepage (as well as about page) isn't optimized for mobile screens. though it works absolutely fine for about page, the search box included in it for homepage searches goes away on scrolling in mobile devices. I would suggest making a floating search button for mobile screens, or to make the search box in navigation bar sticky on top as we scroll through it, or have a collapsible fixed navigation bar for mobile screens. Other solutions for it are also welcome.

Let's keep this issue open for all, and @piechowiakmichal please label the issue with tags like `HacktoberFest`, `good-first-issue`, `up-for-grabs` to enhance its discoverability.

**Comment by piechowiakmichal** [comment]
Good idea. Can also make the search bar on the About Us page redirect you to the results on the Home page. The issue is opened, thanks!

**Comment by AkhilChoubey** [comment]
I would like to work on this ..

**Comment by piechowiakmichal** [comment]
I assigned you, thanks.

**Comment by piechowiakmichal** [comment]
Create a pull request, I'll linked them here.

**Comment by AkhilChoubey** [comment]
Ok , I am on it..

**Comment by piechowiakmichal** [comment]
The issue remains open.

**Comment by piechowiakmichal** [comment]
If you want to solve this issue, please write how you want to do it first - information for everyone.

**Comment by SurendarSingh** [comment]
I will work on it @piechowiakmichal, I will solve with the bootstrap navbar classes which can solve your need like collapsible fixed navigation bar for mobile screens.

**Comment by piechowiakmichal** [comment]
Hi @SurendarSingh! Can you make a sticky search bar instead?

**Comment by SurendarSingh** [comment]
Yeah ok

**Comment by piechowiakmichal** [comment]
It would also be nice if a search with about us brings you to the results on the main page, but of course I do not require it.

**Comment by piechowiakmichal** [comment]
I assigned you, thank you!

**Comment by SurendarSingh** [comment]
@piechowiakmichal Hey, look into the changes and do let me know your response and changes if required.

**Comment by SurendarSingh** [comment]
Ok, wait I will re-work on it

**Comment by SurendarSingh** [comment]
@piechowiakmichal Hey, Now is it fine?
If you want me to make any changes, do let me know, I will do it
